### PR TITLE
Publish all Resources and AOs by default

### PR DESCRIPTION
**Pull request by marktriggs**
https://www.pivotaltracker.com/story/show/123569271

### CODE REVIEW

File: src/converters/resource_converter.rb [hunk 1]
```diff
@@ -229,6 +231,7 @@ def build_notes(collection, db)
           'jsonmodel_type' => 'note_singlepart',
           'type' => 'custodhist',
           'content' => [collection[:history]],
+          # THINKME: Is this really meant to be unpublished?
           'publish' => false,
```
Comment (marktriggs) [inline src/converters/resource_converter.rb]: @jambun Do you have any memory of why this note type is set to unpublished?

Comment (jambun) [inline src/converters/resource_converter.rb]: None. It was probably just 'there'

Comment (marktriggs) [inline src/converters/resource_converter.rb]: Cool.  Amended the PR to knock that out too.

**Comment by jambun** [comment]
Might need a rebase ...

**Comment by marktriggs** [comment]
Also fixed!

**Pull request merged**